crates/rooch-framework-tests/src/binding_test.rs

```diff
@@ -73,24 +72,17 @@ impl RustBindingTest {
             MoveOSStore::mock_moveos_store_with_data_dir(moveos_db_path.as_path())?;
         let rooch_store = RoochStore::mock_rooch_store(rooch_db_path.as_path())?;
         let sequencer = AccountAddress::ONE.into();
-        let gas_schedule_blob = bcs::to_bytes(&default_gas_schedule())
-            .expect("Failure serializing genesis gas schedule");
 
         let genesis = RoochGenesis::build(
-            RoochChainID::LOCAL.genesis_ctx(sequencer, gas_schedule_blob),
```

Comment:
Maybe I have misunderstood. If we don't save the gas_schedule_blob in GenesisContext, how can we call the init() function in genesis.move to initialize the gas_entries with onchain data?

Reply:
here

https://github.com/rooch-network/rooch/blob/71705dc2c82d543aa2059e0b2dbc2cd570c07fbe/crates/rooch-genesis/src/lib.rs#L176-L182